source/installation/install_from_source.rst

```diff
@@ -25,8 +25,7 @@ Step 1 - Download Software
 
 For obvious reasons, you need to download rsyslog. Here, I assume that
 you use a distribution tarball. If you would like to use a version
-directly from the repository, see `build rsyslog from
-repository <build_from_repo.html>`_ instead.
```

Comment:
same as above

Reply:
This was a broken link. In current docs we shouldn't be referencing reST docs via a .html extension.

Reply:
you are right! I overlooked the html